database/index.js

```diff
@@ -1,15 +1,17 @@
 const mongoose = require('mongoose');
-const fs = require('fs');
-const {Schema} = require('mongoose');
-const products = require('./data.js');
+const { Schema } = require('mongoose');
 
+
+// Retain schema and model for queries to proper collection in db
 const productSchema = new Schema({
   'prime_pic': String,
-  image1: String,
-  image2: String,
+  'pic_1': String,
+  'pic_2': String,
 });
 const Product = mongoose.model('Product', productSchema);
+
 const connectionString = 'mongodb+srv://tbay:tbay@tbay-pdiop.gcp.mongodb.net/tbay?retryWrites=true&w=majority';
-mongoose.connect(connectionString, {useNewUrlParser: true, useUnifiedTopology: true});
+mongoose.connect(connectionString, { useNewUrlParser: true, useUnifiedTopology: true });
 const db = mongoose.connection;
-Product.insertMany(products);
+
+// Product.insertMany(products);
```

Comment:
I like how your naming conventions are succinct and are similar to each other.
Line 17 has a bit of commented out code, are you planning on using this later?